src/OpenLoco/GameCommands/ChangeCompanyColour.cpp

```diff
@@ -0,0 +1,76 @@
+#include "../Audio/Audio.h"
+#include "../CompanyManager.h"
+#include "../GameException.hpp"
+#include "../OpenLoco.h"
+#include "../Ui/WindowManager.h"
+#include "../Ui/WindowType.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    // 0x0043483D
+    static uint32_t changeCompanyColour(uint8_t flags, uint8_t type, bool toggleMode, CompanyId_t companyId, bool isSecondary, uint8_t value)
+    {
+        GameCommands::setExpenditureType(ExpenditureType::Miscellaneous);
+        GameCommands::setPosition({ static_cast<int16_t>(0x8000), 0, 0 });
+
+        auto* company = CompanyManager::get(companyId);
+
+        if ((flags & Flags::apply) == 1)
```

Comment:
I dont think the == 1 adds anything here. I can perhaps see why you would want == 0 for the inverse but == 1 does not make sense.